nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/PutHBaseCell.java

```diff
@@ -82,6 +79,8 @@ protected PutFlowFile createPut(final ProcessSession session, final ProcessConte
         final String columnQualifier = context.getProperty(COLUMN_QUALIFIER).evaluateAttributeExpressions(flowFile).getValue();
         final String timestampValue = context.getProperty(TIMESTAMP).evaluateAttributeExpressions(flowFile).getValue();
 
+        final String visibilityStringToUse = pickVisibilityString(columnFamily, columnQualifier, flowFile, context);
```

Comment:
When I was configured the processor like below:
- Row Identifier: ${hbase.rowid}
- Column Family: ${hbase.column.family}
- Column Qualifier: ${hbase.column.qualifier}
I was looking for a property for visibility expression to specify EL similarly, e.g. '${hbase.visibility}'. Currently, in order to add visibility, a FlowFile must have an attribute named 'visibility.family.qualifier', or a dynamic property is needed. I think I wanted to put the configuration directory without adding a dynamic property. But it should be fine since we now have a detailed docs. Thanks, no change is needed for the comment.